client/visualizer/src/main/sidebar.ts

```diff
@@ -301,8 +319,13 @@ export default class Sidebar {
       case Mode.QUEUE:
         this.innerDiv.appendChild(this.matchqueue.div);
         break;
+      case Mode.PROFILER:
+        this.innerDiv.append(this.profiler.div);
+        break;
     }
 
-    // this.cb();
+    if (this.cb !== undefined) {
```

Comment:
check this

Reply:
this.cb() is set to update the gamearea content based on the currently open tab. I guess this was commented out after removing the map editor (currently the replay viewer is always shown) and I added it back in in order to show the speedscope iframe when the Profiler tab is active. Not sure about the `!== undefined` check, but I'm guessing it caused an error without it.

Reply:
Makes sense, thanks